feature/extrasA/src/main/resources/modules/system/layers/fuse/com/rometools/rome/1.0/module.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module xmlns="urn:jboss:module:1.1" name="com.rometools.rome" slot="1.0">
```

Comment:
Is this an intended deletion?

Reply:
I 

> feature/extrasA

rome 1.0 is still there, but not in feature/extrasA. Maybe the removal should have been a part of some previous commit.

Reply:
We can probably get this change in and follow up on that issue later